### PR TITLE
Implemented some functionality to the guess command

### PR DESCRIPTION
**Pull request by JSteve0**
Added guess command to cli
Created guess method with most functionality
Added tests

relates to #16

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -30,4 +33,11 @@ vorpal
     callback();
   });
 
+vorpal
```
Comment (mharrison7787) [inline src/app.js:36]: Even if its self explanatory I think we should add comments anyway so we get good grades.

File: src/app.js [hunk 2]
```diff
@@ -6,6 +6,9 @@ var vorpal = require('vorpal')();
 //GameManager object
 var GameManager = new (require('./classes/GameManager.js'))();
```
Comment (mharrison7787) [inline src/app.js:8]: Go more detail of what the variable is doing. I know its self explanatory but just in case.

**Review by mharrison7787 — CHANGES_REQUESTED** [review]
I added some inline comments that you should address before merging. Just so the whole group can understand the code

File: src/app.js (hunk 2 above)
Comment (JSteve0) [reply src/app.js:8]: Added more detail and resolved.

File: src/app.js (hunk 1 above)
Comment (JSteve0) [reply src/app.js:37]: Added more documentation.

**Comment by JSteve0** [comment]
> I added some inline comments that you should address before merging. Just so the whole group can understand the code

Fixed those issues with a new commit, thanks for the feedback!

**Review by mharrison7787 — APPROVED** [review]
Looks great now that each definition has at least one comment. Thanks for adding these.

**Comment by mharrison7787** [comment]
Thanks for fixing this. Looks good now with the comments in each file

**Pull request merged**